src/citrine/informatics/analysis_configuration.py

```diff
@@ -6,7 +6,30 @@
 
 
 class CrossValidationAnalysisConfiguration(Serializable['CrossValidationAnalysisConfiguration']):
-    """Configuration settings for running CV analysis in a Performance Workflow."""
+    """Configuration settings for running cross-validation in a performance workflow.
+
+    Parameters
+    ----------
+    name : str
+        Name of the analysis configuration
+    description: str
+        Description of the analysis configuration
+    n_folds: int
+        Number of k-folds
+    n_trials: int
+        Number of k-fold cross-validation trials to run
```

Comment:
Since "k" is not defined, I suggest "Number of cross-validation trials to run, each with `n_folds` folds."